packages/canvas-tokens-docs/stories/system/examples/Depth.tsx

```diff
@@ -1,33 +1,49 @@
 import * as React from 'react';
 import {system} from '@workday/canvas-tokens-web';
-import {TokenGrid} from '../../../components/TokenGrid';
+import {TokenGrid, formatJSVar} from '../../../components/TokenGrid';
 
 interface DepthToken {
-  label: string;
+  /** The name of the CSS variable */
+  cssVar: string;
+  /** The formatted name of the JS variable */
+  jsVar: React.ReactNode;
+  /** The actual string value of the token */
   value: string;
 }
 
-const depthTokens: DepthToken[] = Object.values(system.depth).map(varName => {
+const depthTokens: DepthToken[] = Object.entries(system.depth).map(([level, varName]) => {
   const value = getComputedStyle(document.documentElement).getPropertyValue(varName);
 
   return {
-    label: varName,
+    cssVar: varName,
+    jsVar: formatJSVar(`system.depth.${level}`),
     value: value.split('),').join('),\n'),
   };
 });
 
 export function DepthTokens() {
   return (
-    <TokenGrid caption="depth tokens" headings={['Sample', 'Name', 'Value']} rows={depthTokens}>
+    <TokenGrid
+      caption="depth tokens"
+      headings={['Sample', 'CSS Variable', 'JS Variable', 'Values']}
+      rows={depthTokens}
+    >
       {token => (
         <>
           <TokenGrid.RowItem>
             <TokenGrid.Sample style={{boxShadow: token.value}} />
           </TokenGrid.RowItem>
           <TokenGrid.RowItem>
-            <TokenGrid.MonospaceLabel>{token.label}</TokenGrid.MonospaceLabel>
+            <TokenGrid.MonospaceLabel>{token.cssVar}</TokenGrid.MonospaceLabel>
+          </TokenGrid.RowItem>
+          <TokenGrid.RowItem>
+            <TokenGrid.MonospaceLabel>{token.jsVar}</TokenGrid.MonospaceLabel>
+          </TokenGrid.RowItem>
+          <TokenGrid.RowItem>
+            {token.value.split('),').map((item, i) => (
+              <span key={i}>{i === 0 ? `${item})` : item}</span>
```

Comment:
`!i` here instead of `i === 0`? there is no way `i` will be 0 or positive number